assets/icon/map_locator.svg

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
a recommendation: try using svgo to optimize this svg file.
In projects where there is more svg files, create a script that optimizes all svg files automatically.